micro_manager/micro_manager.py

```diff
@@ -331,11 +351,11 @@ def _initialize(self) -> None:
 
         (
             self._mesh_vertex_ids,
-            mesh_vertex_coords,
+            self._mesh_vertex_coords,
```

Comment:
Is this change related to this PR?

Reply:
Yes,  `self._mesh_vertex_coords` is used a few times in the crash handling to log where the simulation crashed and it is used for the interpolation.

Reply:
Okay got it :+1: